### PR TITLE
Added map with home position / craft position / flight trail #490

### LINKED ISSUE #490: Idea Map on Viewer
Hallo, is it posbile to implement a map to the viewer. So maybe on the left top cornver that you can see a map where you actually flight if you have a gps signal. And then you can have a zoom by speed or a fixed zoom. Maybe with google maps.

### PR DESCRIPTION
**Pull request by bonchan**
This PR resolves #490 
image
The map uses gps data to display home position, craft position and flight trail 
It can be configured to show altitude colored trail or single color trail
image

PR open for discussion.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: yarn-error.log
Comment (haslinghuis) [inline yarn-error.log:1]: Please add to .gitignore:
```
npm-debug.log*
yarn-debug.log*
yarn-error.log*
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: yarn-error.log
Comment (bonchan) [reply yarn-error.log]: done

**Review by haslinghuis — DISMISSED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
